docs/pages/blog/turbo-1-2-0.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Turborepo 1.2
+date: 2022/04/06
+description: Turborepo 1.2 introduces improved task filtering, artifact signing and integrity, human-readable and JSON dry runs, and more!
+tag: web development
+ogImage: /images/blog/turbo-1-2-0/twitter-card.png
+---
+
+# Turborepo 1.2
+
+import { Authors } from '../../components/Authors'
+
+<Authors authors={['jaredpalmer', 'becca__z', 'gaspargarcia_', 'gsoltis']} />
+
+Since releasing Turborepo v1.1 in late January, we've seen incredible adoption and community growth:
+
+- 6.5k+ GitHub Stars
+- 140k+ weekly NPM downloads (doubling since our [last blog post for v1.1](/blog/turbo-1-1-0))
+- 90+ OSS contributors
+- 900+ members of the [Turborepo Community Discord](https://turborepo.org/discord)
+- 1.6 years of Time Saved through Remote Caching on Vercel, saving more than 2.5 months every week
```

Comment:
```suggestion
- **140k+** weekly NPM downloads (doubling since our [last blog post for v1.1](/blog/turbo-1-1-0))
- **90+** OSS contributors
- **900+** members of the [Turborepo Community Discord](https://turborepo.org/discord)
- **1.6 years** of Time Saved through Remote Caching on Vercel, saving more than 2.5 months every week
```